snakemake_executor_plugin_kubernetes/__init__.py

```diff
@@ -57,6 +57,26 @@ class ExecutorSettings(ExecutorSettingsBase):
             "when using Google Cloud GKE Autopilot."
         },
     )
+    privileged: Optional[bool] = field(
+        default=False,
+        metadata={
+            "help": "This creates a privileged container which allows to "
+            "mount storage inside the running container."
+        },
+    )
+    persistent_volume_claim_name: Optional[str] = field(
+        default=None,
+        metadata={
+            "help": "Mount the PVC with said name inside container in "
+            "`persistent_volume_claim_path` location"
+        },
+    )
+    persistent_volume_claim_path: Optional[str] = field(
+        default=None,
+        metadata={
+            "help": "Mount the PVC inside container in this location"
+        },
+    )
```

Comment:
Maybe we should allow for multiple PVCs to be mounted?

Reply:
What about a setting `persistent_volumes` with a syntax like `<name>:<path>` and nargs="+" instead of the way you propose here?

Reply:
Could you update this PR accordingly? This way, we avoid another breaking change in a future update.